drivers/serial/uart_mcux_flexcomm.c

```diff
@@ -1146,6 +1158,40 @@ static int mcux_flexcomm_init_common(const struct device *dev)
 	return 0;
 }
 
+#if UART_IS_WAKEUP
+static void mcux_flexcomm_pm_prepare_wake(const struct device *dev, enum pm_state state)
+{
+	const struct mcux_flexcomm_config *config = dev->config;
+	struct mcux_flexcomm_data *data = dev->data;
+	USART_Type *base = config->base;
+
+	/* Switch to the lowest possible baud rate, in order to
+	 * both minimize power consumption and also be able to
+	 * potentially wake up the chip from this mode.
+	 */
+	if (pm_policy_device_is_disabling_state(dev, state, 0)) {
+		clock_control_configure(config->clock_dev, config->lp_clock_subsys, NULL);
+		data->old_brg = base->BRG;
+		data->old_osr = base->OSR;
+		base->OSR = 8;
+		base->BRG = 0;
```

Comment:
How did you decide on these values. A comment would be helpful.

Reply:
There already is a comment on line 1168-1171

Reply:
True, but how we calculate 8 and 0?

Reply:
There is no calculation involved, BRG is the baud rate generator register and I put 0 which is the min, and OSR is the clocks per bit register which I put 8 which is the max